src/Text/Pandoc/Writers/Docx.hs

```diff
@@ -256,8 +257,29 @@ writeDocx opts doc@(Pandoc meta _) = do
                        )
 
   -- styles
+  let lang = case lookupMeta "lang" meta of
+               Just (MetaInlines [Str s]) -> Just s
```

Comment:
Let's also match on `MetaString s` -- if they use the `--metadata` flag on the command line, we'll have a `MetaString`.